### ISSUE TITLE
depricated getIO will break heartbeat

### ISSUE THREAD

**Issue by ivan1986** [php-amqplib/php-amqplib#696]
If processing message get long time need to send heartbeat to server, for single thread apps.
For now simple function
 ```
    public function pingHeartbeat(): void
    {
        $this->connection->getIO()->check_heartbeat();
    }
```
 works, but getIO is depricated, 
```    /**
     * @return \PhpAmqpLib\Wire\IO\AbstractIO
     * @deprecated
     */
    public function getIO()
    {
        return $this->io;
    }
```
please add function to connection interface for correct work of this case

**Comment by ramunasd** [comment]
You can use `$connection->select(null);` instead.

**Comment by ivan1986** [comment]
@ramunasd ok, but it's strange
i think need method and example for long run, but if you disagree just close issue

**Comment by ramunasd** [comment]
I will add method for heartbeat check without access to IO driver itself.

**Comment by ramunasd** [comment]
fixed in #697 will be released in version 2.10.0
